doc/api/child_process.md

```diff
@@ -2254,12 +2256,12 @@ if the child process could not be successfully spawned.
 added: v0.7.10
 -->
 
-By default, the parent will wait for the detached child to exit. To prevent the
-parent from waiting for a given `subprocess` to exit, use the
+By default, the parent will wait for the detached child process to exit.
+To prevent the parent from waiting for a given `subprocess` to exit, use the
```

Comment:
```suggestion
By default, the parent process will wait for the detached child process to exit.
To prevent the parent process from waiting for a given `subprocess` to exit, use the
```